### PR TITLE
Cloud July 2022 Release Notes

### PR DESCRIPTION
**Pull request by gemma-shay**
https://cockroachlabs.atlassian.net/browse/DOC-4619

### CODE REVIEW

**Review by jramos-crl — APPROVED** [review]
We are cutting it close with the storage limitations removal but we feel confident that it should be out by next week / this release notes. I will let you know @gemma-shay if that isn't the case - so for now it looks good to me and we can proceed as if the change will be live by then.

**Review by gemma-shay — COMMENTED** [review]
Good catch, will add.

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/14373#-:-N5epkf18sh-rPgR5a83:bi6drqi)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @emily-horing, @JeffSwenson, @kevin-v-ngo, and @thtruo)

___
*[`releases/cloud.md` line 16 at r1](https://reviewable.io/reviews/cockroachdb/docs/14373#-N5bC2_V6lvOS9b6elCN:-N5epCv2-As7oUQadZzP:brzzfc7) ([raw file](https://github.com/cockroachdb/docs/blob/a2f7a11a11d27f517e604f649425c0b01e87305b/releases/cloud.md#L16)):*
<details><summary><i>Previously, emily-horing wrote…</i></summary><blockquote>

I think by the 6th we expect to be running 22.1.2 as well on serverless -- cc @jeffswenson to confirm
</blockquote></details>

Gotcha, will update optimistically and can change back before publishing if not.
___
*[`releases/cloud.md` line 23 at r1](https://reviewable.io/reviews/cockroachdb/docs/14373#-N5bCEWt7PPDaaf4Cw3a:-N5ep1zh4LW4kyrQcEOU:b-3o73vb) ([raw file](https://github.com/cockroachdb/docs/blob/a2f7a11a11d27f517e604f649425c0b01e87305b/releases/cloud.md#L23)):*
<details><summary><i>Previously, emily-horing wrote…</i></summary><blockquote>

I wasn't aware of this, what drove this change/where did it come from?
</blockquote></details>

https://github.com/cockroachlabs/managed-service/pull/8899

**Review by emily-horing — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/14373)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @emily-horing, @gemma-shay, @kevin-v-ngo, and @thtruo)

___
*[`releases/cloud.md` line 23 at r1](https://reviewable.io/reviews/cockroachdb/docs/14373#-N5bCEWt7PPDaaf4Cw3a:-N5jgaKR3GTT6FzrUxI9:b-afo50t) ([raw file](https://github.com/cockroachdb/docs/blob/a2f7a11a11d27f517e604f649425c0b01e87305b/releases/cloud.md#L23)):*
<details><summary><i>Previously, gemma-shay (Gemma Shay) wrote…</i></summary><blockquote>

https://github.com/cockroachlabs/managed-service/pull/8899
</blockquote></details>

ha i didn't even know we did that. cool.

**Comment by gemma-shay** [comment]
@kevin-v-ngo @thtruo @ianjevans hoping to get this out tomorrow just fyi!

**Review by thtruo — APPROVED** [review]
Obs Infra and KV Obs don't have new additions to call out. FWIW what you listed here LGTM 

> Reviewed 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/14373#-:-N6ElWrc3YBnxbqE40Be:b9fcb7e)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @emily-horing, @gemma-shay, @ianjevans, and @kevin-v-ngo)

File: releases/cloud.md [hunk 1]
```diff
@@ -13,9 +13,24 @@ Get future release notes emailed to you:
...
+As of July 6, 2022, {{ site.data.products.serverless }} clusters are running CockroachDB [v21.2.12](v21.2.html) and new {{ site.data.products.dedicated }} clusters are running CockroachDB [v22.1.2](v22.1.html).
 {{site.data.alerts.end}}
 
+## July 6, 2022
+
+<h3>Console changes</h3>
+
+- The [**Connect to your cluster**](../{{site.versions["stable"]}}/connect-to-the-database.html) dialog now includes code snippets for [supported languages and tools](../{{site.versions["stable"]}}/third-party-database-tools.html).
+- The [**Connect to your cluster**](../cockroachcloud/connect-to-a-serverless-cluster.html) dialog for clusters running CockroachDB [v22.1](v22.1.html) now loads more quickly.
+- The **Databases** page now shows the recommended number of indexes for the database.
```
Comment (kevin-v-ngo) [inline releases/cloud.md:25]: We should remove "The **Databases** page now shows the recommended number of indexes for the database."

I'm assuming this refers to "Drop unused index" recommendations which is not available until 22.2.

**Review by kevin-v-ngo — CHANGES_REQUESTED** [review]

File: releases/cloud.md (hunk 1 above)
Comment (kevin-v-ngo) [reply releases/cloud.md:25]: Adding @THardy98 to correct me here

Comment (THardy98) [reply releases/cloud.md:25]: Correct, initially it was merged for 22.1, but after adding the table/index details pages on 22.2, we decided to version-gate everything to 22.2 to keep everything together.

**Review by ianjevans — APPROVED** [review]

**Pull request merged**